src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,11 +2,16 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    public static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
         //TODO: add corresponding log message about method login was called
+        logger.info("Method login was called for user with login: {}", login);
```

Comment:
```suggestion
        logger.debug("Method login was called for user with login: {}", login);
```

for calling methods it's better to use debug level